lib/Conf.ml

```diff
@@ -1401,7 +1401,9 @@ module Operational = struct
   let debug =
     let doc = "Generate debugging output." in
     Decl.flag ~default ~names:["g"; "debug"] ~doc ~kind
-      (fun conf elt -> update conf ~f:(fun f -> {f with debug= elt}))
+      (fun conf elt ->
+        if elt.v then Box_debug.enable_stacktraces := true ;
```

Comment:
This means that enabling the debug output implies enabling the stacktraces. Is the purpose to have the debugging output without having the stacktraces ?

Reply:
Enabling the stacktraces is a bit costly in terms of performances, and they have to be collected early. This allows to disable them when they are not going to be used, that is when there is no debug run that is going to be performed.

Reply:
There's already a bool variable for that: `Fmt.debug`. According to what you said, `enable_stacktraces` is unnecessary.

Reply:
No, fmt_debug is set much later

Reply:
Box_debug.debug is set by the following function :
```
val with_box_debug : t -> t
```

When this function is called, the `t` you give as input is already generated, so its stack traces are either in memory or they are lost forever.

Reply:
Makes sense then!

Reply:
Though, to have clearer dependencies between the modules, I'd prefer if that variable was set from `Translation_unit`.